js/renovation/ui/scroll_view/topPocket.tsx

```diff
@@ -44,6 +60,8 @@ export const viewFunction = (viewModel: TopPocket): JSX.Element => {
 })
 
 export class TopPocket extends JSXComponent<TopPocketProps>() {
+  @InternalState() _state = TopPocketState.STATE_RELEASED;
```

Comment:
By default, the state value equals to the STATE_RELEASED value
https://github.com/DevExpress/DevExtreme/blob/21_1/js/renovation/ui/scroll_view/_toDelete/ui.scroll_view.native.pull_down.js#L56

Reply:
I don't remove this line, because the _releaseState method is called from multiple methods: 
first from render and the second - from the _stateReleased (handleScroll is initiator)